src/components/dialogs/ConfirmDissolution.vue

```diff
@@ -13,11 +13,14 @@
         {{getModalTitle}}
         </p>
         <p class="warning-text" id="dialog-text">
-          You are about to voluntarily dissolve <strong>{{ getEntityName }}</strong>;
+          You are about to {{ dissolutionType }} <strong>{{ getEntityName }}</strong>;
           once this process is completed and the required documents are filed, the {{ entityTitle }} will be
           struck from the register and dissolved, ceasing to be an incorporated {{ subEntityTitle }} under the
           {{ entityAct }} Act.
         </p>
+        <p class="warning-text" id="dialog-text">
```

Comment:
put inside v-if so, p tag wont be there when there is no additional label

Reply:
resolved